packages/theming/android-theme/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/android-theme",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Don't manually bump your own package versions. Beachball takes care of that for us. Doing it manually messes with our NPM publish pipeline, like mentioned https://github.com/microsoft/fluentui-react-native/issues/651

Reply:
```
ERROR: Attempting to bump to a version that already exists in the registry: 
@fluentui-react-native/android-theme@0.2.0
Something went wrong with the publish! Manually update these package and versions
```

Updated them manually because of this [log message](https://dev.azure.com/ms/ui-fabric-react-native/_build/results?buildId=151371&view=logs&j=745f91f2-6a7c-581b-9fbc-0f5efa6695e3&t=3ada3ce3-cad9-58da-eaa5-0574e0880da0&l=53) and couldn't find reason for failure.

Reply:
oooh I see. I think it would be better to do that in a separate review rather that just updates the packages, but whatever works I guess. I'm not sure who set up the beachball publishing for us, I think it was @tkamireh ? He might have better insight on how to fix this

Reply:
Fixed  in [#656](https://github.com/microsoft/fluentui-react-native/pull/656) by initiating a change in the package.